easybuild/easyconfigs/o/OpenMM/OpenMM-7.1.1-intel-2017b-Python-3.6.3.eb

```diff
@@ -0,0 +1,52 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = "CMakeMake"
+
+name = "OpenMM"
+version = "7.1.1"
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://simtk.org/home/openmm'
+description = """OpenMM is a toolkit for molecular simulation."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+toolchainopts = {'opt': True}
+
+source_urls = ['https://github.com/pandegroup/openmm/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['ba915d1bc2ea8643c699e48fa4386646dc799a7b4f165350ff1317d8c71a1493']
+
+builddependencies = [
+    ('CMake', '3.9.5'),
+]
+
+separate_build_dir = True
+
+dependencies = [
+    ('Python', '3.6.3'),
+    ('SWIG', '3.0.12', '-Python-%(pyver)s'),
+]
+
+runtest = 'test -e ARGS="-E \'(Integrator)|(Thermostat)|(Barostat)|(Rpmd)\'"'
```

Comment:
@tovrstra Can you try wrapping this in triple quotes, and maybe with a space at the end of the string to keep it readable?

```python
runtest = """test -e ARGS="-E \'(Integrator)|(Thermostat)|(Barostat)|(Rpmd)\"' """